.gitignore

```diff
@@ -158,3 +158,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# CDK
```

Comment:
@emileten Let's add pre-commit hooks for `flake8` `whitespace` and `black` and `isort` at a minimum.

Reply:
done https://github.com/developmentseed/cdk-eoapi/pull/1/commits/cac6aa328dcead87eb373491ef23000d914dcf44. [I set up `pre-commit.ci` so that it runs in this repository.](https://pre-commit.ci/)